### PR TITLE
razorpay refactor readme

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Files need to be reviewed
-Major files imports are rectified (You can review them)
-Files that are added new

Files need not to be reviewed
-CSS files and any backend file 
-Database files

Thanks for reviewing this PR

### CODE REVIEW

File: src/AllRoutes.jsx [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
-import { ProfileCard } from "components/DifferentCards/ProfileCard/ProfileCard";
-import { PrivateRoute } from "components/CustomRoutes/PrivateRoute";
-import { RestrictedRoute } from "components/CustomRoutes/RestrictedRoute";
+import {
+  RestrictedRoute,
+  PrivateRoute,
+  ProfileCard,
+  ProfileAddress,
+} from "components";
+import {
```
Comment (nayyyhaa) [inline src/AllRoutes.jsx:9]: Appreciation: great that you've refactored and created imports for pages & components.

File: src/components/DifferentCards/AddressCard/AddressCard.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const AddressCard = ({ address ,state,setState}) => {
+  return (
+    <div className="address-card__wrapper">
+      <p className="text-2 ">
+        <input type="radio" 
+        checked={address.name===state.name}
+        onChange={()=>{setState(()=>address)}}
+        /> {address.name}
+      </p>
+      <p className="">
```
Comment (nayyyhaa) [inline src/components/DifferentCards/AddressCard/AddressCard.jsx]: you can remove className if no class is applied.

File: src/components/navigation/NavBar.jsx [hunk 3]
```diff
@@ -75,48 +48,71 @@ function NavBar({ menuBtn = false }) {
...
+                </div>
+              </li>
+            ) : (
+              <>
+                <li className="nav-bottom__list-item change-nav-display">
+                  <Link to="/products-page" className="btn nav-bottom-btn text">
+                    Turbo-chargers
+                  </Link>
+                </li>
+                <li className="nav-bottom__list-item change-nav-display">
```
Comment (nayyyhaa) [inline src/components/navigation/NavBar.jsx:74]: suggestion: you can create separate data files and loop around them to avoid code repetition.

**Review by nayyyhaa — APPROVED** [review]
Nice work Akshat. Code is well-formatted + entire refactoring is done. You can merge the changes. 💯

File: src/components/DifferentCards/CartProductBig/NormalCard.jsx [hunk 4]
```diff
@@ -1,15 +1,14 @@
...
-
 import { Link, useNavigate, useLocation } from "react-router-dom";
 import { useDispatch, useSelector } from "react-redux";
 import {
   addToCart,
   deleteItemFromWishlist,
   postItemToWishlist,
-} from "./../../../Redux/Reducers-Redux/operationsSlice";
-// Small product Card
+} from "../../../Redux/Reducers-Redux/operationsSlice";
```
Comment (Developers-codz) [inline src/components/DifferentCards/CartProductBig/NormalCard.jsx]: can use absolute path instead of relative path

**Review by Developers-codz — APPROVED** [review]
The code looks clean and readable, you can merge 💯

**Review by VLeads — APPROVED** [review]
Checked your code. It all looks good. Go ahead !!

File: src/pages/ProductsPage/ProductsPage.jsx [hunk 5]
```diff
@@ -1,35 +1,64 @@
...
+import ReactPaginate from "react-paginate";
+const PER_PAGE = 6;
 const ProductsPage = () => {
   const { filtered } = useFilterManger();
+  const [currentPage, setCurrentPage] = useState(0);
+  const handlePageClick = ({ selected: selectedPage }) => {
+    setCurrentPage(selectedPage);
+  };
+  const offset = currentPage * PER_PAGE;
+  const currentPageData = filtered.sorted
```
Comment (VLeads) [inline src/pages/ProductsPage/ProductsPage.jsx:15]: Appreciate you are using pagination

**Review by VLeads — APPROVED** [review]

File: src/AllRoutes.jsx (hunk 1 above)
Comment (Dev-AkshatGupta) [reply src/AllRoutes.jsx:9]: Thanks for reviewing the PR

File: src/components/DifferentCards/AddressCard/AddressCard.jsx (hunk 2 above)
Comment (Dev-AkshatGupta) [reply src/components/DifferentCards/AddressCard/AddressCard.jsx]: Thanks for noticing it

File: src/components/DifferentCards/CartProductBig/NormalCard.jsx (hunk 4 above)
Comment (Dev-AkshatGupta) [reply src/components/DifferentCards/CartProductBig/NormalCard.jsx]: Yeah done it thanks a lot for reviewing pr

File: src/pages/ProductsPage/ProductsPage.jsx (hunk 5 above)
Comment (Dev-AkshatGupta) [reply src/pages/ProductsPage/ProductsPage.jsx:15]: Thanks for reviewing the pr

**Pull request merged**